index.html

```diff
@@ -1816,37 +1816,39 @@ <h3>Lifecycle Details</h3>
 
         <ol>
           <li>
-The <a>verifier</a> defines its policies for accepting
+A <a>verifier</a> defines its own policies for accepting
 <a>verifiable credentials</a> from <a>holders</a> for its supported actions.
           </li>
           <li>
-The <a>issuer</a> issues a <a>verifiable credential</a> to the <a>holder</a>.
+An <a>issuer</a> issues one or more <a>verifiable credentials</a> to 
+a <a>holder</a>.
           </li>
           <li>
-The <a>holder</a> might pass on its <a>verifiable credentials</a> to another
-<a>holder</a>.
+A <a>holder</a> might pass on one or more of its <a>verifiable credentials</a> 
+to another <a>holder</a>.
           </li>
           <li>
-The final <a>holder</a> presents its <a>verifiable credentials</a> to the
-<a>verifier</a> in a <a>verifiable presentation</a>, requesting a supported
+A <a>holder</a> presents one or more of its <a>verifiable credential(s)</a> 
+to a <a>verifier</a> in a <a>verifiable presentation</a>, requesting a supported
```

Comment:
This has to say optionally since VPs are optional not mandatory. Therefore change to

A <a>holder</a> presents one or more of its <a>verifiable credential(s)</a> 
to a <a>verifier</a>, optionally in a <a>verifiable presentation</a>, requesting a supported